modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -18608,6 +18607,7 @@ this-field-requires-a-valid-style-value=This field requires a valid style value.
 this-field-type-does-not-support-translations=This field type does not support translations.
 this-file-contains-headers=This File Contains Headers
 this-file-extension-is-not-supported-by-docusign=This file extension is not supported by DocuSign.
+this-file-is-too-large-to-preview.-the-maximum-size-for-preview-files-is-x=This file is too large to preview. The maximum size for preview files is {0}.
```

Comment:
You know I'm not a native English speaker but I would say that the sentence is not correct. As I understand the max file size is for the original file: when the original file is less or equal to this max file size then a preview is generated. Therefore, the last part of the sentence is not correct:

* _The maximum size for preview files is X_ means that "preview files" have a maximum size of X (not true)
* This can be fixed by replacing "for preview files" with "for previewing files" or "to preview files".

What do you think @austinchiang ?

Reply:
Well, I'm awful on english too so I tend to copy what is put on the story/figma :D

Please, if is not correct, I will change it.

Reply:
I'm not sure about this :( 

The preview size is not involved here.
Is the size of the file we will make the preview...


This modify anything?

Reply:
I think there is enough context from the previous sentence ("This file is too large to preview") to infer that the maximum size is referring to the size of the file. But if we want to make it more explicit then maybe something like "The maximum file size for previews is {0}"

Reply:
thanks @austinchiang